app/src/layout/Objective.tsx

```
@@ -71,12 +72,17 @@ export const Objective: React.FC<ObjectiveProps> = (props) => {
-                <CircleHelp className="h-5 w-5 hover:text-orange-500 hover:cursor-pointer" />
-              </PopoverTrigger>
-              <PopoverContent>{objective.description}</PopoverContent>
-            </Popover>
+            <>
+              <CircleHelp
+                className="h-5 w-5 hover:text-orange-500 hover:cursor-pointer"
+                onClick={() => setModalOpen(true)}
+              />
+              {modalOpen && (
+                <Modal title="Objective Details" setIsOpen={() => setModalOpen(false)}>
+                  <div dangerouslySetInnerHTML={{ __html: objective.description }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)